### PR TITLE
devops-219: adding in grype orb

### PR DESCRIPTION
**Pull request by hunguyen1**
Signed-off-by: Hung Nguyen <hung.nguyen@anchore.com>

**What this PR does / why we need it**:
Add the grype orb. anchore orb has been deprecated and will be removed in another PR.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>)(, fixes #<issue_number, ...)` format, will close the issue when PR is merged*: fixes #:
devops-219

**Special notes**:

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -521,15 +517,12 @@ workflows:
...
+      - grype/list-image-vulns:
           context: dockerhub
-          filters: *filter_rc_tags
           requires:
             - push_rc_image
-          image_name: anchore/anchore-engine-dev:${CIRCLE_TAG}
-          private_registry: true
-          timeout: "2000"
-          policy_failure: true
+          image-name: anchore/anchore-engine-dev:rc
```
Comment (Btodhunter) [inline .circleci/config.yml:524]: This can use ${CIRCLE_TAG} for the image tag.

File: .circleci/config.yml [hunk 2]
```diff
@@ -639,15 +632,12 @@ workflows:
...
-          name: anchore_image_scan
+      - grype/list-image-vulns:
           context: dockerhub
           requires:
             - push_dev_image
-          image_name: anchore/anchore-engine-dev:${CIRCLE_SHA1}
-          private_registry: true
-          timeout: "2000"
-          policy_failure: true
+          image-name: anchore/anchore-engine-dev:${CIRCLE_SHA1}
```
Comment (Btodhunter) [inline .circleci/config.yml:639]: This can use ${CIRCLE_TAG} for the image tag.

**Review by Btodhunter — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (hunguyen1) [reply .circleci/config.yml]: updated. thanks!

**Review by Btodhunter — APPROVED** [review]
lgtm

**Review by Btodhunter — CHANGES_REQUESTED** [review]
All env vars used for image tags must use dev instead since the orb cannot take in env vars as a param.

**Comment by hunguyen1** [comment]
> All env vars used for image tags must use dev instead since the orb cannot take in env vars as a param.

Updated and put it in the job as a default param.

**Review by d3vdr3y — APPROVED** [review]
lgtm

**Review by Btodhunter — APPROVED** [review]
lgtm!

**Pull request merged**